contracts/modules/PaymentsImmutables.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.24;
 
 import {IWETH9} from '../interfaces/external/IWETH9.sol';
-import {IAllowanceTransfer} from 'permit2/src/interfaces/IAllowanceTransfer.sol';
+import {IPermit2} from 'permit2/src/interfaces/IPermit2.sol';
```

Comment:
just wondering why. readability?

Reply:
oh we do need IPermit2 to compile so we have it in typechain - so keeping it